invenio_records_rest/views.py

```diff
@@ -74,6 +76,12 @@ def validation_error(error):
         """Catch validation errors."""
         return RESTValidationError().get_response()
 
+    @blueprint.errorhandler(RequestError)
+    def elasticsearch_badrequest_error(error):
+        """Catche problems in ElasticSearch queries."""
+        description = gettext(error.error)
```

Comment:
@lnielsen Ok, I am not using the exception message then. It looks like this right now:
![zenodo_error2](https://cloud.githubusercontent.com/assets/6481083/15738938/fcebf68a-28ad-11e6-8302-bbbe0b97eac4.png)